Resources/Prototypes/Loadouts/Generic/items.yml

```diff
@@ -789,6 +789,18 @@
   items:
     - WhiteCane
 
+- type: loadout
```

Comment:
I talked with TCJ a while back about plushies in loadouts, and the consensus was to not put plushies and other vanity items with no clear purpose to loadouts.

What you can do is put `PlushieShadowkin` in random plushie spawners and crates:
- `ToySpawner` in `Resources/Prototypes/Entities/Markers/Spawners/Random/toy.yml`
- `PresentRandom` in `Resources/Prototypes/Entities/Objects/Decoration/present.yml`
- `AllPlushiesTable` in `Resources/Prototypes/Catalog/Fills/Crates/fun.yml`
- `CrateCargoGambling` in `Resources/Prototypes/Catalog/Fills/Crates/cargo.yml`
- `MailNFPlushie` in `Resources/Prototypes/DeltaV/Entities/Objects/Specific/Mail/mail.yml`